homeassistant/components/concord232/alarm_control_panel.py

```diff
@@ -19,39 +19,46 @@
 DEFAULT_HOST = 'localhost'
 DEFAULT_NAME = 'CONCORD232'
 DEFAULT_PORT = 5007
+DEFAULT_MODE = 'audible'
 
 SCAN_INTERVAL = datetime.timedelta(seconds=10)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_CODE): cv.string,
+    vol.Optional(CONF_MODE, default=DEFAULT_MODE): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
 })
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Concord232 alarm control panel platform."""
     name = config.get(CONF_NAME)
+    code = config.get(CONF_CODE)
+    mode = config.get(CONF_MODE)
     host = config.get(CONF_HOST)
     port = config.get(CONF_PORT)
 
     url = 'http://{}:{}'.format(host, port)
 
     try:
-        add_entities([Concord232Alarm(url, name)], True)
+        add_entities([Concord232Alarm(url, name, code, mode)], True)
     except requests.exceptions.ConnectionError as ex:
         _LOGGER.error("Unable to connect to Concord232: %s", str(ex))
 
 
 class Concord232Alarm(alarm.AlarmControlPanel):
     """Representation of the Concord232-based alarm panel."""
 
-    def __init__(self, url, name):
+    def __init__(self, url, name, code, mode):
         """Initialize the Concord232 alarm panel."""
         from concord232 import client as concord232_client
 
         self._state = None
         self._name = name
+        self._code = code or None
```

Comment:
The `or None` can be removed. `code` is `None` per default.

Reply:
```suggestion
        self._code = code
```

Reply:
I agree!!